### DISCUSSION TITLE
Getting Cannot load representation of type public.image while using FilePicker plugin.

### DISCUSSION THREAD

**Discussion by ginthozan-v** [capawesome-team/capacitor-plugins#61] [Q&A] [answered]
Hi there, 
I am using the FilePicker plugin to upload images and video from gallery. but when I using this I am getting the following error.

`Cannot load representation of type public.image`

```
[Error] Error picking file: 
Error: Cannot load representation of type public.image
CapacitorException — util.js:21
returnResult — user-script:2:768
(anonymous function) — user-script:2:750
Global Code — Script Element 1:1
	error
	(anonymous function) (FileUploader.tsx:58)
```

- Ionic 7
- Capacitor
- React

Can someone help me how to fix it.

**Comment by robingenz** [comment, marked as answer]
This is a known issue in the simulator, see https://github.com/capawesome-team/capacitor-file-picker/issues/82#issuecomment-1476836282.

**Reply by ginthozan-v** [reply]
Hi, Thank you for the reply. appreciate 👍
